eth/contracts/Most.sol

```diff
@@ -2,35 +2,47 @@
 
 pragma solidity ^0.8.20;
 
-import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
-import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
-import "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
-import "@openzeppelin/contracts-upgradeable/access/Ownable2StepUpgradeable.sol";
-import "@openzeppelin/contracts-upgradeable/utils/PausableUpgradeable.sol";
-
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import {Initializable} from "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import {UUPSUpgradeable} from "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
+import {Ownable2StepUpgradeable} from "@openzeppelin/contracts-upgradeable/access/Ownable2StepUpgradeable.sol";
+import {PausableUpgradeable} from "@openzeppelin/contracts-upgradeable/utils/PausableUpgradeable.sol";
+import {IWETH9} from "./IWETH9.sol";
+
+/// @title Most
+/// @author Cardinal Cryptography
 contract Most is
     Initializable,
     UUPSUpgradeable,
     Ownable2StepUpgradeable,
     PausableUpgradeable
 {
+    using SafeERC20 for IERC20;
+
+    /// @dev This amount of gas should be sufficient for ether transfers
+    /// and simple fallback function execution, yet still protecting against reentrancy attack.
+    uint256 constant GAS_LIMIT = 3500;
+
     uint256 public requestNonce;
     uint256 public committeeId;
     address payable public wethAddress;
 
+    mapping(bytes32 from => bytes32 to) public supportedPairs;
+    mapping(bytes32 requestHash => Request) public pendingRequests;
+    mapping(bytes32 requestHash => bool) public processedRequests;
+    /// @dev committeeMemberId = keccak256(abi.encodePacked(committeeId, comitteeMemberAddress))
+    mapping(bytes32 committeeMemberId => bool) private committee;
+    mapping(uint256 committeeId => uint256) public committeeSize;
+    mapping(uint256 committeeId => uint256) public signatureThreshold;
+    // @dev storage gap for future contract upgrades compatibility
+    uint256[50] __gap;
```

Comment:
define constant for __gap size instead of 50

Reply:
According to OZ, it is a convention for reserving "storage slots", allowing new storage variables in future versions of the contract. Source: https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
However, there's also a new convention since OZ 5.x  - [ERC-7201](https://docs.openzeppelin.com/upgrades-plugins/1.x/writing-upgradeable#namespaced-storage-layout)

Reply:
Actually, there aren't any child contracts of "Most" in the design, so we don't have to follow this convention.